src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -105,12 +125,74 @@ _IRQL_requires_max_(DISPATCH_LEVEL)
 _Function_class_(CXPLAT_DATAPATH_RECEIVE_CALLBACK)
 void
 UdpRecvCallback( 
-    _In_ CXPLAT_SOCKET* /* Binding */,
-    _In_ void* /* Context */,
+    _In_ CXPLAT_SOCKET* Binding,
+    _In_ void* Context,
     _In_ CXPLAT_RECV_DATA* RecvBufferChain
     )
 {
+    CXPLAT_RECV_DATA* Datagram;
+    while ((Datagram = RecvBufferChain) != NULL) {
+        RecvBufferChain = Datagram->Next;
+        Datagram->Next = NULL;
+        if (!Datagram->Allocated) {
+            continue;
+        }
```

Comment:
Unnecessary. This flag is only used for asserts (which shouldn't be necessary here).